tools/mmp/linker/MonoMac.Tuner/MonoMacNamespaces.cs

```diff
@@ -113,6 +114,122 @@ public void Process (LinkContext context)
 					// no sample
 					remove_dlopen = !namespaces.Contains (Namespaces.GameKit);
 					break;
+				case Constants.GameControllerLibrary:
```

Comment:
Maybe you could add a dictionary that maps constants and namespaces ? and then loop over it